src/Mvc/src/Microsoft.AspNetCore.Mvc/Microsoft.AspNetCore.Mvc.csproj

```diff
@@ -9,7 +9,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Reference Include="Microsoft.AspNetCore.Mvc.Analyzers" PrivateAssets="None" />
```

Comment:
FYI @pranavkm  - this reference shouldn't be used here. Found it as a result of trying to figure out why analyzers were in the reference API surface for Microsoft.AspNetCore.App.